spec/components/previews/alert_component_preview.rb

```diff
@@ -0,0 +1,41 @@
+class AlertComponentPreview < ViewComponent::Preview
+  # @!group Kitchen Sink
+  def default
+    render(AlertComponent.new(message: 'A default message'))
+  end
+
+  def info
+    render(AlertComponent.new(message: 'An info message', type: :info))
+  end
+
+  def success
+    render(AlertComponent.new(message: 'A success message', type: :success))
+  end
+
+  def warning
+    render(AlertComponent.new(message: 'A warning message', type: :warning))
+  end
+
+  def error
+    render(AlertComponent.new(message: 'An error message', type: :error))
+  end
+
+  def emergency
+    render(AlertComponent.new(message: 'An emergency message', type: :emergency))
+  end
```

Comment:
I haven't seen us use emergency in our app, the glyph looked positioned wrong to me? Should we fix or remove?

<img width="509" alt="Screen Shot 2022-02-22 at 2 04 48 PM" src="https://user-images.githubusercontent.com/458784/155227445-fda8e404-d7ca-49c8-bbcf-08b43079412c.png">

Reply:
The upside of listing out every option is that it surfaces some interesting bugs from lack of use! This looks to be a bug at the design system level, likely because we override the default design system alert icons with [our own set of icons](https://github.com/18F/identity-style-guide/tree/main/src/img/alerts), and since the "emergency" state was added in a later version, we hadn't yet handled it.

Personally I'd be fine to leave it so that we can face our shame head-on.

Reply:
😂  Maybe a follow-up task (and/or design system sync) to account for any unhandled variations/bugs that need design help? CC: @nickttng

Reply:
Welp. This looks like a follow-up task/conversation. To check: Does this kind of bug have any user-facing impact?

Reply:
@nickttng I'm not aware that we show an emergency alert anywhere, so fortunately I don't expect it should impact the end-user.